src/PharoThings-Hardware-Core.package/PotDevice.class/instance/connectTo..st

```diff
@@ -1,5 +1,6 @@
 controlling
 connectTo: aBoard
 	board := aBoard.
+	self configureDefaultPeripherals.
```

Comment:
This does not look right because when user will specify his own configuration it will be reset when device will be installed (reset to default version)
I think Initialize method is enough place to call it

Reply:
Ok, I moved this call to initialize in the driver.